packages/event/src/Charcoal/Event/EventDispatcher.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Charcoal\Event;
+
+use League\Event\EventDispatcher as LeagueEventDispatcher;
+use League\Event\HasEventName;
+use Psr\Log\LoggerAwareTrait;
+
+/**
+ * Event Dispatcher
+ *
+ * Extension of \League\Event\EventDispatcher to add extra features.
+ */
+class EventDispatcher extends LeagueEventDispatcher
+{
+    use LoggerAwareTrait;
+
+    public function dispatch(object $event): object
+    {
+        if ($this->logger) {
+            if ($event instanceof HasEventName) {
+                $this->logger->notice('Event [' . $event->eventName() . '] dispatched', [
```

Comment:
With the event name after "Event dispatched", this log's message format matches the "anonymous" event log. This makes it easier to search for all dispatch logs.

```suggestion
                $this->logger->notice('Event dispatched [' . $event->eventName() . ']', [
```